.devcontainer/Dockerfile.llvm-project

```diff
@@ -0,0 +1,131 @@
+# By default, use ubuntu:focal
+ARG BASE_IMAGE="ubuntu:focal"
+FROM ${BASE_IMAGE}
+
+# Label the image for various checking and cleanup
+ARG LLVM_PROJECT_SHA1="ba8b8a73fcb6b830e63cd8e20c6e13b2a14d69bf"
+LABEL llvm_project_sha1=${LLVM_PROJECT_SHA1}
+
+
+ARG NPROC=16
+ARG WORK_DIR=/workdir
+WORKDIR ${WORK_DIR}
+
+# Install tools needed
+RUN distro=$(cat /etc/os-release|grep -Po '(?<=^ID=").*(?=")|(?<=^ID=)[^"].*[^"]') \
+    && TZ="Europe/Berlin" \
+    && if [ "${distro}" = "debian" ] || [ "${distro}" = "ubuntu" ]; then \
+          DEBIAN_FRONTEND=noninteractive && \
+          apt-get update -qq && \
+          apt-get install -qq -y --no-install-recommends tzdata && \
+          ln -sf /usr/share/zoneinfo/${TZ} /etc/localtime && \
+          dpkg-reconfigure -f noninteractive tzdata && \
+          apt-get install -qq -y --no-install-recommends \
+                  autoconf automake ca-certificates cmake curl \
+                  default-jdk-headless gcc g++ git libncurses-dev \
+                  libtool make maven ninja-build openjdk-11-jdk-headless \
+                  python3 python3-dev python3-distutils python3-numpy \
+                  python3-pip python3-pytest-xdist python3-setuptools \
+                  python3-typing-extensions zlib1g-dev && \
+          rm -rf /var/lib/apt/lists/* && \
+          ln -sf /usr/bin/pytest-3 /usr/bin/pytest; \
+       elif [ "${distro}" = "rhel" ] || [ "${distro}" = "fedora" ]; then \
+          ln -sf /usr/share/zoneinfo/${TZ} /etc/localtime && \
+	  ([ -x /usr/bin/microdnf ] && microdnf install -y yum) && \
+          yum install -q -y \
+              https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
+          yum update -q -y && \
+          yum install -q -y \
```

Comment:
Did this come from their docker? I would say since the base image should always be debian and use the `apt-get` package manager. 

I don't know how much you want to support other distros :D